classes/local/evidenceitems.php

```diff
@@ -150,7 +150,7 @@ public function course_duration_evidence($userid, $courseid, $credentialid, $com
             }
 
             if ($enrolmenttimestamp && $enrolmenttimestamp != 0 &&
-                ($enrolmenttimestamp < $completedtimestamp)) {
+                ($enrolmenttimestamp <= $completedtimestamp)) {
```

Comment:
I think we shouldn't make this change because it's not directly related to the issue but it changes the behavior a bit.

Reply:
This is the method that will call the API client to post the evidence. Without the change on this condition, the new test added won't pass because both dates are the same and it won't try to post the evidence but it should do it, that is the reason I updated this. Should I still revert it?